metadata.json

```diff
@@ -25,26 +25,27 @@
   "issues_url": "https://github.com/puppetlabs/pltraining-classroom/issues",
   "description": "Important! This is not the module you are looking for\n    unless you are a trainer teaching Puppet Labs training courses!\n    - Configures and sets up classroom environment for Puppet Classroom trainings.",
   "dependencies": [
-    {"name":"hunner/wordpress",     "version_requirement":">= 0.6.0"},
-    {"name":"jamtur01/irc",         "version_requirement":">= 0.0.7"},
-    {"name":"puppetlabs/apache",    "version_requirement":">= 1.1.1"},
-    {"name":"puppetlabs/concat",    "version_requirement":">= 1.0.4"},
-    {"name":"puppetlabs/git",       "version_requirement":">= 0.2.0"},
-    {"name":"puppetlabs/haproxy",   "version_requirement":">= 1.0.0"},
-    {"name":"puppetlabs/inifile",   "version_requirement":">= 1.0.0"},
-    {"name":"puppetlabs/mysql",     "version_requirement":">= 2.3.1"},
-    {"name":"puppetlabs/ntp",       "version_requirement":">= 3.2.0"},
-    {"name":"puppetlabs/stdlib",    "version_requirement":">= 4.0.0"},
-    {"name":"puppetlabs/pe_gem",    "version_requirement":">= 0.0.1"},
-    {"name":"puppetlabs/vcsrepo",   "version_requirement":">= 1.1.0"},
-    {"name":"pltraining/rbac",      "version_requirement":">= 0.0.2"},
-    {"name":"pltraining/userprefs", "version_requirement":">= 0.0.3"},
-    {"name":"rismoney/chocolatey",  "version_requirement":">= 0.0.3"},
-    {"name":"zack/exports",         "version_requirement":">= 0.0.4"},
-    {"name":"zack/r10k",            "version_requirement":">= 2.3.0"},
-    {"name":"jriviere/windows_ad"  ,"version_requirement":">= 0.2.0"},
-    {"name":"badgerious/windows_env"  ,"version_requirement":">= 2.2.1"},
-    {"name":"jordan/fileshare"  ,"version_requirement":">= 1.1.1"},
-    {"name":"puppetlabs/windows"  ,"version_requirement":">= 1.0.0"}
+    {"name":"hunner/wordpress",               "version_requirement":">= 0.6.0"},
+    {"name":"jamtur01/irc",                   "version_requirement":">= 0.0.7"},
+    {"name":"puppetlabs/apache",              "version_requirement":">= 1.1.1"},
+    {"name":"puppetlabs/concat",              "version_requirement":">= 1.0.4"},
+    {"name":"puppetlabs/git",                 "version_requirement":">= 0.2.0"},
+    {"name":"puppetlabs/haproxy",             "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/inifile",             "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/mysql",               "version_requirement":">= 2.3.1"},
+    {"name":"puppetlabs/ntp",                 "version_requirement":">= 3.2.0"},
+    {"name":"puppetlabs/stdlib",              "version_requirement":">= 4.0.0"},
+    {"name":"puppetlabs/pe_gem",              "version_requirement":">= 0.0.1"},
```

Comment:
if we ever install `eyaml` or the like--something with a CLI component--then we'll need it. Might be good to keep around.